prefect_kubernetes/pods.py

```diff
@@ -0,0 +1,361 @@
+"""Module for interacting with Kubernetes pods from Prefect flows."""
+
+from typing import Any, Callable, Dict, Optional, Union
+
+from kubernetes.client.exceptions import ApiException
+from kubernetes.client.models import V1DeleteOptions, V1Pod, V1PodList
+from kubernetes.watch import Watch
+from prefect import task
+from prefect.utilities.asyncutils import run_sync_in_worker_thread
+
+from prefect_kubernetes.credentials import KubernetesCredentials
+
+
+@task
+async def create_namespaced_pod(
+    kubernetes_credentials: KubernetesCredentials,
+    body: V1Pod,
```

Comment:
There most likely be a better parameter name that `pod_config`, but that's what came to mind for me.
```suggestion
    pod_config: V1Pod,
```